chain/default_store.go

```diff
@@ -61,24 +60,21 @@ type DefaultStore struct {
 
 	// Tracks tipsets by height/parentset for use by expected consensus.
 	tipIndex *TipIndex
-
-	// TODO block cache should go here
 }
 
 // Ensure DefaultStore satisfies the Store interface at compile time.
 var _ Store = (*DefaultStore)(nil)
 
 // NewDefaultStore constructs a new default store.
 func NewDefaultStore(ds repo.Datastore, stateStore *hamt.CborIpldStore, genesisCid cid.Cid) *DefaultStore {
-	bs := bstore.NewBlockstore(ds)
-	priv := hamt.CborIpldStore{Blocks: bserv.New(bs, offline.Exchange(bs))}
+	priv := bstore.NewBlockstore(ds)
 	return &DefaultStore{
-		privateStore: &priv,
-		stateStore:   stateStore,
-		headEvents:   pubsub.New(128),
-		ds:           ds,
-		tipIndex:     NewTipIndex(),
-		genesis:      genesisCid,
+		bsPriv:     priv,
```

Comment:
Made sense to make the private store for holding types.Block an ipfs blockstore

Reply:
I made a somewhat arbitrary decision based on poor understanding.  There was nothing significant about it.